src/ExposureHistoryContext.tsx

```diff
@@ -89,6 +88,16 @@ const ExposureHistoryProvider: FunctionComponent<ExposureHistoryProps> = ({
     })
   }, [])
 
+  const getExposureStatus = useCallback(async () => {
+    const exposureStatus = await StorageUtils.getNewExposuresData()
+    setUserHasNewExposure(exposureStatus)
+  }, [])
+
+  const setExposureStatus = useCallback(async (exposureStatus: boolean) => {
```

Comment:
```suggestion
  const setHasExposure = useCallback(async (hasExposure: boolean) => {
```

Reply:
why are we setting the exposure status from the JS layer? isnt this data that is determined by the native layer?

Reply:
Slightly related to the above, but this is because this isn't determining whether there was an exposure, but rather whether the user has seen that exposure. I think your suggestion about changing the naming of the variable could make this more clear.